tutorials/monitoring/3-configuring-endpoints/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Monitoring NServiceBus solutions: Configuring endpoints"
+reviewed: 2017-10-10
+summary: Setting up NServiceBus endpoints to be monitored by the Particular Service Platform.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Throughput and processing time"
+  nextUrl: tutorials/monitoring/4-endpoint-performance
+---
+
+include: monitoring-intro-paragraph
+
+include: monitoring-sample-solution
+
+This third lesson guides you through setting up all of the endpoints in your system to be monitored by the Particular Software Platform.
+
+
+## Configure endpoints
```

Comment:
How about we state that one is using the sample solution e.g. Billing is a good place to start and add comments in `Program.cs` where say: "this is the place where you should put x-y-z"